src/products/firefox_desktop/firefox_l10n_faqs.md

```diff
@@ -18,6 +18,26 @@ Shipping a feature in English for everyone is not a good option either: as an En
 
 In the end, both options are a product decision, and the localization team can help assess the impact on users and community. How to ship a feature only to en-US users depends on several factors (technology, information available at the time the feature is loaded, etc.), and the best approach is to get in touch with the [engineering part of the Localization team](https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=66651657) (aka “tech team”).
 
+## Builds and distribution
+
+### How can users get localized builds?
+
+Unlike mobile products, where we ship multilingual builds by default and users can simply switch language from the app’s settings, for Firefox there are several ways to get a localized build, independently from the platform:
```

Comment:
While this applies to Android mobile, it does not for iOS: you'll have to choose the language under the OS settings. There is no in-app switcher for iOS Firefox products

Reply:
Ah, thanks for taking a look. I'll fix it, although I'm not sure this is going to land as is at this point 👍🏼